less/index.less

```diff
@@ -52,8 +55,202 @@ table {
 * {
     box-sizing: border-box;
 }
+// **************************************
+.HPB {       // hot pink border
+	border: 2px solid hotpink;
+}
+
+.DBB {       // dodgerblue border
```

Comment:
Thanks for creating comments here. An alternative would be to use names that don't require it.